catalog.yml

```diff
@@ -1,3 +1,6 @@
+- repo_url: https://github.com/laurentperrinet/2023-01-31_formes-et-perception
+  html_url: https://laurentperrinet.github.io/2023-01-31_formes-et-perception/
+  thumbnail_url: https://raw.githubusercontent.com/laurentperrinet/2023-01-31_formes-et-perception/main/content/images/eVscope-20230319-220133_crop-small.png
```

Comment:
preferable to set thumbnail in [metadata.yaml](https://github.com/laurentperrinet/2023-01-31_formes-et-perception/blob/main/content/metadata.yaml) rather than here (such that it can be used as the preview image for any social sharing).